modules/multi-architecture-modify-machine-set-aws.adoc

```diff
@@ -4,28 +4,30 @@
 
 :_mod-docs-content-type: PROCEDURE
 [id="multi-architecture-modify-machine-set-aws_{context}"]
+= Adding a multi-architecture compute machine set to your AWS cluster
 
-= Adding an ARM64 compute machine set to your cluster
+After creating a multi-architecture cluster, you can add nodes with different architectures.
 
-To configure a cluster with multi-architecture compute machines, you must create a AWS ARM64 compute machine set. This adds ARM64 compute nodes to your cluster so that your cluster has multi-architecture compute machines.
+Following are the examples of adding multi-architecture compute machines to a multi-architecture cluster:
```

Comment:
```suggestion
You can add the following multi-architecture compute machines to a multi-architecture cluster:

*  Adding 64-bit x86 compute machines to a cluster that uses 64-bit ARM control plane machines and already includes 64-bit ARM compute machines. In this case, 64-bit x86 is considered the secondary architecture.
* 64-bit ARM compute machines. You can add 64-bit ARM compute machines  to a cluster that uses 64-bit x86 control plane machines and already includes 64-bit x86 compute machines. In this case, 64-bit ARM is considered the secondary architecture.
* 64-bit ARM compute machines. You can add 64-bit ARM compute machines to a cluster that uses 64-bit x86 control plane machines and already includes 64-bit x86 compute machines. In this case, 64-bit ARM is considered the secondary architecture.
```

Reply:
Updated as follows:

"You can add multi-architecture compute machines to a multi-architecture cluster in the following ways:"